apps/kitchensink-react/src/DocumentCollection/DocumentEditorRoute.tsx

```diff
@@ -40,9 +36,7 @@ function Editor() {
   const {data: name = ''} = useDocument({...docHandle, path: 'name'})
   const setName = useEditDocument({...docHandle, path: 'name'})
 
-  const [value, setValue] = useState('')
-
-  const {data: document} = useDocument({...docHandle})
+  const {data: document} = useDocument(docHandle)
   const setDocument = useEditDocument(docHandle)
```

Comment:
`docHandle` should be spread to these ;)

```suggestion
  const {data: document} = useDocument({...docHandle})
  const setDocument = useEditDocument({...docHandle})
```

Reply:
Most of the hooks (including this one) now expect the handle to be spread. Previously they were expecting the document handle *object* as a param, now they expect the document handle's *properties* as params.

Reply:
@ricokahler Can explain more if needed!

Reply:
Oh my gosh, you're so right, sorry! Only needs to be spread if we're also providing options. Thanks for bearing with me 😅